src/vip-framework/checks/checkIsolatedPoolsComptrollers.ts

```diff
@@ -145,7 +144,11 @@ const runPoolTests = async (pool: PoolMetadata) => {
   await comptroller.connect(timelockSigner).setPriceOracle(originalOracle);
 };
 
-export const checkIsolatedPoolsComptrollers = (): void => {
+// NOTE: The default supplier for each pool will be VTreasury, if in case VTreasury has no
+//       underlying token balance in any or only in the last market in the list, you can use the
+//       poolToSupplierMap in order to specify a supplier per pool
+//       You can check multisig/simulations/vip-010/vip-010-ethereum/simulations.ts for example.
+export const checkIsolatedPoolsComptrollers = (poolToSupplierMap: { [comptroller: string]: string } = {}): void => {
```

Comment:
I'd go with a named argument so that the intent is more evident at the call site:

```
interface CheckIsolatedPoolsComptrollersParams {
  poolToSupplierMap?: { [comptroller: string]: string }
}

export const checkIsolatedPoolsComptrollers = (poolToSupplierMap: CheckIsolatedPoolsComptrollerParams): void => { ... }
```

Otherwise it looks like we're rewriting comptrollers